src/index.js

```diff
@@ -0,0 +1,945 @@
+      shiftUp: 'b',
+      shiftDown: 'B',
+    },
+    ru: {
+      shiftUp: 'и',
+      shiftDown: 'И',
+    },
+  },
+  {
+    class: ['KeyN'],
+    eng: {
+      shiftUp: 'n',
+      shiftDown: 'N',
+    },
+    ru: {
+      shiftUp: 'т',
+      shiftDown: 'Т',
+    },
+  },
+  {
+    class: ['KeyM'],
+    eng: {
+      shiftUp: 'm',
+      shiftDown: 'M',
+    },
+    ru: {
+      shiftUp: 'ь',
+      shiftDown: 'Ь',
+    },
+  },
+  {
+    class: ['Comma'],
+    eng: {
+      shiftUp: ',',
+      shiftDown: '<',
+    },
+    ru: {
+      shiftUp: 'б',
+      shiftDown: 'Б',
+    },
+  },
+  {
+    class: ['Period'],
+    eng: {
+      shiftUp: '.',
+      shiftDown: '>',
+    },
+    ru: {
+      shiftUp: 'ю',
+      shiftDown: 'Ю',
+    },
+  },
+  {
+    class: ['Slash'],
+    eng: {
+      shiftUp: '/',
+      shiftDown: '?',
+    },
+    ru: {
+      shiftUp: '.',
+      shiftDown: ',',
+    },
+  },
+  {
+    class: ['ArrowUp', 'special'],
+    eng: {
+      shiftUp: '▲',
+      shiftDown: '▲',
+    },
+    ru: {
+      shiftUp: '▲',
+      shiftDown: '▲',
+    },
+  },
+  {
+    class: ['ShiftRight', 'special'],
+    eng: {
+      shiftUp: 'Shift',
+      shiftDown: 'Shift',
+    },
+    ru: {
+      shiftUp: 'Shift',
+      shiftDown: 'Shift',
+    },
+  },
+];
+
+const keyRow5 = [
+  {
+    class: ['ControlLeft', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+  {
+    class: ['MetaLeft', 'special'],
+    eng: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+    ru: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+  },
+  {
+    class: ['AltLeft', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['Space', 'special'],
+    eng: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+    ru: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+  },
+  {
+    class: ['AltRight', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['ArrowLeft', 'special'],
+    eng: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+    ru: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+  },
+  {
+    class: ['ArrowDown', 'special'],
+    eng: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+    ru: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+  },
+  {
+    class: ['ArrowRight', 'special'],
+    eng: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+    ru: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+  },
+  {
+    class: ['ControlRight', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+];
+
+/* global document, localStorage */
```

Comment:
Same situation as was for previous tasks. How this comment help me to understand your code?

Reply:
Left this comment because of ESLint (https://eslint.org/docs/rules/no-undef)